packages/devtools_app/lib/src/memory/memory_controller.dart

```diff
@@ -1032,6 +1032,12 @@ class MemoryController extends DisposableController
     return allocations;
   }
 
+  /// If viewing offline data (Android collected) the connection may not be Android.
```

Comment:
dart doc: new line after the first sentence

Reply:
Done.